code/src/App.js

```diff
@@ -1,12 +1,22 @@
-import React from 'react'
-import data from './data.json'
+import React from "react"
+import data from "./data.json"
+import { Album } from "./components/Album"
+import { Artists } from "./components/Artist"
+import { Header } from "./components/Header"
+import { tsPropertySignature } from "@babel/types"
 
 console.log(data)
+console.log(Album)
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <div className='card'>
+      <Header />
+      <section className='album-container'>
+        {data.albums.items.map((item) => {
+          return <Album key={item.id} albumTitle={item.name} image={item.images[1].url} artists={item.artists[0].name} urlArtistName={item.artists[0].external_urls.spotify} albumUrl={item.external_urls.spotify} />
+        })}
```

Comment:
Regarding the artists.
Since some of the albums have multiple artists you should make sure to show all of them.
Pass along the whole artists-array as a prop, so instead of `artists={item.artists[0].name} urlArtistName={item.artists[0].external_urls.spotify}` go for ` artists={album.artists}`